api/discord/controllers/discord.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const axios = require('axios');
+
+module.exports = {
+  async discordAuth(ctx){
+    const { code } = ctx.query;
+    if (!code) {
+      return ctx.send({ message: 'There is not a code in the url' }, 500);
```

Comment:
Nit: To be more accurate, you are looking at the query parameter of the URL.

Reply:
You mean instead of sending a message "There is not a code in the url", we send "There is not a query in the url"?